sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/WalletBirthdayExt.kt

```diff
@@ -0,0 +1,37 @@
+package cash.z.ecc.android.sdk.internal
+
+import cash.z.ecc.android.sdk.type.WalletBirthday
+import org.json.JSONObject
+
+// Version is not returned from the server, so version 1 is implied.  A version is declared here
+// to structure the parsing to be version-aware in the future.
+internal val WalletBirthday.Companion.VERSION_1
```

Comment:
This might be overdoing it.

Shall we adopt TreeState Protobuf as it is? maybe we can test that we can create a protobuf from a JSON file and use it as the underlying storage. 

We need to rename the checkpoints because we are falling off the Protocol Buffers conventions which are thought to avoid this kind of issues. 

If we fold back to using TreeState, this and the checkpoint renaming won't be needed at all. And we will be always looking after the protobuf definitions instead.